main.py

```diff
@@ -131,6 +131,61 @@ def calculate_interest(self, rate: float) -> None:
         print("Debit accounts do not earn interest.")
 
 
+class SavingsAccount:
```

Comment:
We could have some unit tests within this commit to make sure that changes are correct